pyccel/codegen/pipeline.py

```diff
@@ -220,7 +220,7 @@ def handle_error(stage):
                                  accelerator=accelerator,
                                  includes=())
     elif fflags is not None:
-        fflags = fflags.strip().split(' ')
+        fflags = [f for f in fflags.strip().split(' ') if f]
```

Comment:
Isn't it sufficient to use `split` without any argument? That is
```python
fflags = fflags.split()
```
According to the [documentation](https://docs.python.org/3/library/stdtypes.html#str.split)
> If sep is not specified or is None, a different splitting algorithm is applied: runs of consecutive whitespace are regarded as a single separator, and the result will contain no empty strings at the start or end if the string has leading or trailing whitespace

Reply:
Done in cd131e5